### ISSUE TITLE
70 to 100 in 20 seconds

### ISSUE THREAD

**Issue by kevinz000** [tgstation/tgstation#27471]
21:58:59 [0x20057f1] (113,66,1) || the supermatter crystal has reached the emergency point for the first time.
22:01:21 [0x20057f1] (113,66,1) || the supermatter crystal has exploded.
22:01:21 [0x20057f1] (113,66,1) || the supermatter crystal has been destroyed.
Close it if you think this isn't a bug but I think there's something wrong with the supermatter going from "GET AWAY IM ABOUT TO BLOW" to "Engineering and the surrounded areas no longer exist" within 20 seconds.

**Comment by WJohn** [comment]
I think I've noticed this a few times. The supermatter jumping up really, REALLY quickly.

**Comment by Kromgar** [comment]
Oh i have a secret method that does this its very simple. I break the cooling loop from space. 

Guaranteed supermatter explosion in one minute

**Comment by ohnopigeons** [comment]
Atmos-based engine feature

**Comment by Tokiko1** [comment]
I've added damage hardcap defines just for this kind of balance concern. Feel free to lower them if you think you can survive the "HUGBOX!!!" flood of complaints.

**Comment by Tokiko1** [comment]
By the way, this is almost certainly related to the fact that you guys added it to the air subsystem. It probably processes absurdly fast compared to before now.

**Comment by Tokiko1** [comment]
Alternative fix suggestion: On line 224 just before the comment about capping damage, add damage = damage * 0.25. Probably replace the 0.25 with a define like "ATMOS_DAMAGE_SCALING"

Adjust the define DAMAGE_HARDCAP to 0.0025

Most certainly needs testing if it actually works.

**Comment by Armhulen** [comment]
yeah it went from 6% to 100% in seconds

**Comment by fizzle7** [comment]
> Oh i have a secret method that does this its very simple. I break the cooling loop from space.
 Guaranteed supermatter explosion in one minute

Just had that happen to me, someone casually strolled into space, broke the space loop and it went from 30K to 4000K in about 10 seconds or so, by the time I fixed the space loop the freon was out cause I wrenched it in without noticing the space loop. After that the waste pipes filter to cooling pipes stopped working for some fucking reason and the gases werent cycling so the SM ended up exploding, which might be a bug on its own but I'm not sure enough to report it. 

This is really fucking annoying, add rwalls around the space loop at least, cause any fucklet asking the AI for eva can fuck up the SM 2 minutes into the round every fucking round.
